frontend/src/components/common/AuthModal/utils.ts

```diff
@@ -0,0 +1,36 @@
+import { LoginHook, useLogin } from '@/hooks/useLogin';
+import { SignUpHook, useSignUp } from '@/hooks/useSignUp';
+
+interface AuthModalConfig {
+  helperText: string;
+  primaryButtonText: string;
+  secondaryButtonText: string;
+  finishHandler: SignUpHook | LoginHook;
+}
+
+export const signUpConfig: AuthModalConfig = {
+  helperText: 'If you already have an account please.',
+  primaryButtonText: 'Sign up',
+  secondaryButtonText: 'Login',
+  finishHandler: useSignUp,
+};
+
+export const loginConfig: AuthModalConfig = {
+  helperText: "If you don't have an account yet you can sign up.",
+  primaryButtonText: 'Login',
+  secondaryButtonText: 'Register',
+  finishHandler: useLogin,
+};
+
+export const style = {
```

Comment:
do we need this if we're gonna use styled components?

Reply:
Being used as a prop for a MUI component. So probs yes.

Reply:
And possibly temporary as well